### ISSUE TITLE
ranger crash

### ISSUE THREAD

**Issue by electricface** [ranger/ranger#2268]
#### Runtime Environment

- Operating system and version: debian 10
- Terminal emulator and version: alacritty 0.7.2
- Python version: Python 3.7.3
- Ranger version/commit: ranger 1.9.3
- Locale: zh_CN.UTF-8

#### Current Behavior
crash

#### Expected Behavior
Don't crash

#### Steps to reproduce
1. run command trash to delete a directory.

#### Traceback

```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/ranger/core/main.py", line 201, in main
    fm.loop()
  File "/usr/lib/python3/dist-packages/ranger/core/fm.py", line 383, in loop
    ui.handle_input()
  File "/usr/lib/python3/dist-packages/ranger/gui/ui.py", line 266, in handle_input
    self.handle_key(key)
  File "/usr/lib/python3/dist-packages/ranger/gui/ui.py", line 196, in handle_key
    elif not DisplayableContainer.press(self, key):
  File "/usr/lib/python3/dist-packages/ranger/gui/displayable.py", line 275, in press
    focused_obj.press(key)
  File "/usr/lib/python3/dist-packages/ranger/gui/widgets/console.py", line 188, in press
    self.type_key(key)
  File "/usr/lib/python3/dist-packages/ranger/gui/widgets/console.py", line 214, in type_key
    self._answer_question(answer)
  File "/usr/lib/python3/dist-packages/ranger/gui/widgets/console.py", line 197, in _answer_question
    callback(answer)
  File "/usr/lib/python3/dist-packages/ranger/config/commands.py", line 759, in _question_callback
    self.fm.execute_file(files, label='trash')
  File "/usr/lib/python3/dist-packages/ranger/core/actions.py", line 459, in execute_file
    filenames = [f.path for f in files]
  File "/usr/lib/python3/dist-packages/ranger/core/actions.py", line 459, in <listcomp>
    filenames = [f.path for f in files]
AttributeError: 'str' object has no attribute 'path'
```

**Comment by toonn** [comment]
Known issue which I think is fixed on master. Could you try that out?

**Comment by electricface** [comment]
I tested the master branch and it still crashes.
```
ranger
ranger version: ranger-master
Python version: 3.7.3 (default, Nov 27 2020, 09:27:44) [GCC 8.3.0]
Locale: zh_CN.UTF-8
Current file: '/home/del0/projects/build-dde-session-shell-unknown-Debug'

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/ranger/core/main.py", line 203, in main
    fm.loop()
  File "/usr/local/lib/python3.7/dist-packages/ranger/core/fm.py", line 383, in loop
    ui.handle_input()
  File "/usr/local/lib/python3.7/dist-packages/ranger/gui/ui.py", line 276, in handle_input
    self.handle_key(key)
  File "/usr/local/lib/python3.7/dist-packages/ranger/gui/ui.py", line 206, in handle_key
    elif not DisplayableContainer.press(self, key):
  File "/usr/local/lib/python3.7/dist-packages/ranger/gui/displayable.py", line 272, in press
    focused_obj.press(key)
  File "/usr/local/lib/python3.7/dist-packages/ranger/gui/widgets/console.py", line 189, in press
    self.type_key(key)
  File "/usr/local/lib/python3.7/dist-packages/ranger/gui/widgets/console.py", line 215, in type_key
    self._answer_question(answer)
  File "/usr/local/lib/python3.7/dist-packages/ranger/gui/widgets/console.py", line 198, in _answer_question
    callback(answer)
  File "/usr/local/lib/python3.7/dist-packages/ranger/config/commands.py", line 760, in _question_callback
    self.fm.execute_file(files, label='trash')
  File "/usr/local/lib/python3.7/dist-packages/ranger/core/actions.py", line 430, in execute_file
    filenames = [f.path for f in files]
  File "/usr/local/lib/python3.7/dist-packages/ranger/core/actions.py", line 430, in <listcomp>
    filenames = [f.path for f in files]
AttributeError: 'str' object has no attribute 'path'

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues

```

**Comment by toonn** [comment]
Ah #1871 hasn't been merged yet.

**Comment by toonn** [comment]
Fixed by #1871
